src/Fixer/PhpUnit/PhpUnitTestAnnotationFixer.php

```diff
@@ -296,10 +296,14 @@ private function getDocBlockIndex(Tokens $tokens, $index)
      */
     private function hasTestPrefix($functionName)
     {
-        if (!$this->startsWith('test', $functionName) || strlen($functionName) < 5) {
+        if (!$this->startsWith('test', $functionName)) {
             return false;
         }
 
+        if (4 === strlen($functionName)) {
```

Comment:
what we do for <4 now ?

Reply:
`abc` becomes `testAbc` IIUC

Reply:
We won’t get this far if `$functionName` doesn’t start with `test`, see condition above.